src/Perpetuum/Zones/ZoneConfiguration.cs

```diff
@@ -104,7 +104,7 @@ public sealed class ZoneConfiguration
         public int RaceId { get; set; }
         public string Note { get; set; }
 
-        public string ListenerAddress { get; set; }
+        public string ListenerAddress { get; set; } = "127.0.0.1";
```

Comment:
Does setting a default value here change the behavior of a zone configuration?

Reply:
It doesn't affect anything except some debug cases